k8s/namespaces/unspec/kustomization.yaml

```diff
@@ -4,3 +4,4 @@ namespace: unspec
 bases:
 - namespace.yaml
 # Warning : Adding a file here, adds to all environments to which you add your kustomization.
+- unspec-service.yaml
```

Comment:
this adds it to prod as well which I don't think is what you want, you should just be adding it to the environment specific ones for now

Reply:
It is not added to https://github.com/hmcts/cnp-flux-config/blob/master/k8s/prod/common-overlay/kustomization.yaml and being first app, i thought its fine.

Reply:
ah sure